### ISSUE TITLE
Side drawer sluggish

### ISSUE THREAD

**Issue by lesliechw** [quasarframework/quasar-template-meteor#8]
Hello,

Thanks for this Quasar Meteor integration boilerplate.

I just tried run it on Android device. Both of the side drawer seems very sluggish when open and close. Even with swipe slowly open it or close it, the background dimming/darken seems to be flickering. 

Am I doing anything wrong? Please advice, thank you.

Lesz

**Comment by mwarren2** [comment]
Hi, 

The Android devices I've used have all worked very nicely.
The only problems I originally had, and which were fixed very quickly, were on IOS (Safari for mobile is not an easy platform).

However if you want to take the issue further, start a bug report at the main quasar-framework site, along with detailed info about your Android version number, device name and so on, and your problem.

I am only involved with setting up quasar with meteor, I don't have the experience or knowledge necessary for bug fixes to the framework itself.

So I am closing this as I can't personally fix problems like this. However I have always found that bugs have been fixed very quickly by the framework's creator.
